Pop-Grupa-A-Frontend/src/app/registration/duck/registrationTypes.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
Czy jest to na pewno wymagane skoro element jest puty?

Reply:
w poradnikach jest chyba konwencja żeby robić folder duck z rzeczami z reduxa i pewnie temu to zostao stworzone.